indico/modules/events/registration/fields/simple.py

```diff
@@ -62,21 +68,39 @@ def default_value(self):
             return None
         return next((x['id'] for x in data['radioitems'] if x['caption'] == default_item), None)
 
+    def calculate_price(self, registration_data):
+        data = registration_data.field_data.versioned_data
+        for radioitem in data['radioitems']:
```

Comment:
item = next((x for x in data['radioitems'] if registration_data.data == x['id'] and x['is_billable']), None)
    return item['price'] if item else 0

Currently you return a None price if you don't find anything in the loop. Probably not what you want?